spec/services/usps_in_person_proofing/proofer_spec.rb

```diff
@@ -201,6 +219,30 @@ def expect_facility_fields_to_be_present(facility)
       ).to eq(1)
     end
 
+    context 'when the user is going through EIPP' do
+      let(:usps_eipp_sponsor_id) { 'fake-fake' }
```

Comment:
this is a non-integer value so it will get `.to_i` of `0`? if we're going to cast as an integer, can we add an integer?